src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactory.java

```diff
@@ -45,4 +46,11 @@
      * @return a client that is ready to use
      */
     BitbucketUsernameClient getUsernameClient();
+
+    /**
+     * Returns a client that can return the supported web hook events
```

Comment:
Webhook is a single word, so we should change all the occurrence of `web` `hook` to `webhook`